dev/fattest.simplicity/src/componenttest/topology/utils/ServerFileUtils.java

```diff
@@ -0,0 +1,111 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package componenttest.topology.utils;
+
+import java.io.File;
+import java.nio.file.Files;
+
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.topology.impl.LibertyFileManager;
+import componenttest.topology.impl.LibertyServer;
+
+/**
+ * Expands imported files within a Liberty server xml.
+ * The server configs (that include "imports" are assumed to be found in:
+ * publish/servers/<server>/configs/
+ * These config files will have imports located in
+ * publish/shared/config/ of a FAT project
+ *
+ * @author chrisc
+ *
+ */
+public class ServerFileUtils {
+
+    protected static Class<?> thisClass = ServerFileUtils.class;
+    private final String serverConfigDir = "configs";
+    private final String importConfigDir = "config";
+    private final String serverBackupDir = "serverConfigBackups";
+
+    public String expandAndBackupCfgFile(LibertyServer server, String cfgFile) throws Exception {
+        return expandAndBackupCfgFile(server, cfgFile, null);
+    }
+
+    public String expandAndBackupCfgFile(LibertyServer server, String cfgFile, String inTestName) throws Exception {
+
+        String fixedCfgFileName = cfgFile;
+        if (cfgFile == null) {
+            throw new Exception("Requested configuration file can not be null");
+        } else {
```

Comment:
Putting the rest of this inside of an `else` block isn't necessary since the `if` block will throw an exception.

Reply:
removed else.